tests/commands/test_build.py

```diff
@@ -328,7 +332,11 @@ def test_politeexec_stderr_logged(caplog):
 
     cmd = [sys.executable, '-c', "import sys; print('weird, huh?', file=sys.stderr)"]
     polite_exec(cmd)
-    assert [":: weird, huh?"] == [rec.message for rec in caplog.records]
+    expected = [
+        "Running external command " + str(cmd),
+        ":: weird, huh?",
```

Comment:
no distinguishing of stdout vs stderr?

Reply:
Not yet, we still need to do that in general for the tool.